misc/maintenance/fix_cpplint_whitespace.sh

```diff
@@ -3,11 +3,11 @@
 
 # Let's run a set of in-place modifications by sed-commands,
 for file in $@; do
-  sed -i 's/; \/\//;  \/\//' $file # '; //' -> ';  //'
-  sed -i 's/{ \/\//{  \/\//' $file # '{ //' -> '{  //'
-  sed -i 's/} \/\//}  \/\//' $file # '} //' -> '}  //'
-  sed -i 's/for(/for (/' $file     # 'for(' -> 'for ('
-  sed -i 's/if(/if (/' $file       # 'if(' -> 'if ('
-  sed -i 's/\s\s*$//' $file        # 'remove white-space at the end of lines'
+  perl -i -pe 's/; \/\//;  \/\//' $file # '; //' -> ';  //'
+  perl -i -pe 's/{ \/\//{  \/\//' $file # '{ //' -> '{  //'
+  perl -i -pe 's/} \/\//}  \/\//' $file # '} //' -> '}  //'
+  perl -i -pe 's/for(/for (/' $file     # 'for(' -> 'for ('
+  perl -i -pe 's/if(/if (/' $file       # 'if(' -> 'if ('
+  perl -i -pe 's/[ \t]+$//' $file       # 'remove white-space at the end of lines'
```

Comment:
We are not changing this file, but generally, prefer single pass over the file. The substututions can be separated with the `;`, like
```
perl -e 's/foo/bar/g; s/baz/quux/; '
```
as many as you want to do over every line.